requirements/customer/product-environment.tex

```diff
@@ -0,0 +1,46 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}
+      \begin{itemize}
+      \item A connection to a LAN with a central server.
+      \item 2 MB of storage space and an additional 100 MB per image.
+      \end{itemize}
+    \subsubsection{Software Requirements}
+      \begin{itemize}
+      \item Python 3.
+      \item Docker.
+    \end{itemize}
+
+  \subsection{Central Server}
+    \subsubsection{Hardware Requirements}  
+    \begin{itemize}
+      \item A connection to a LAN.
+      \item 500 MB of storage space.
+      \item 2 GB of RAM.
+      \item One physical CPU core.
+    \end{itemize}
+    \subsubsection{Software Requirements}  
+    \begin{itemize}
+      \item Linux Ubuntu 18.04 or newer, or CentOS 7.5 or newer.
+      \item MySQL Database Server version 8.0 or newer.
+      \item Linux kernel version 3.10 or higher.
+      \item CS Docker engine 1.13, EE Deamon 17.03 and higher.
+      \item Python 3.
+    \end{itemize}
+
+  \subsection{Working Machine Client}
+    \subsubsection{Hardware Requirements}
+      In addition to the hardware required to run a job, our worker client would need the following resources:
+    \begin{itemize}
+      \item A connection to a LAN.
+      \item 4 GB of RAM.
```

Comment:
What are you basing the 4 GB RAM requirement on?

Reply:
I assumed that the worker client acts as a worker node. I am basing the hardware requirements on the Docker docs: https://docs.docker.com/v17.09/datacenter/ucp/2.1/guides/admin/install/system-requirements/
There, it says that worker nodes require 4 GB RAM.

Reply:
I see.
That web page doesn't say how the 4 GB RAM figure is calculated.
I'm assuming the 4 GB RAM figure includes the RAM needed for docker containers to run.
I think the amount of RAM required to run the actual Docker engine is less than 1 GB.